index.html

```diff
@@ -336,6 +336,14 @@ <h4 class="panel-title">Field
                         <input class="btn-input calc-trigger" type="checkbox" id="friendGuardR" /><label class="btn btn-xxwide" for="friendGuardR">Friend Guard</label>
                     </div>
                 </div>
+                <div class="btn-group gen-specific g8">
+                    <div class="left" title="Is this Pokemon protected by an ally's Power Spot?">
```

Comment:
```suggestion
                    <div class="left" title="Is this Pokemon boosted by an ally's Power Spot?">
```
Unless I've grossly misunderstood the ability, should this not say boosted rather than protected? Looks like a c/p error from Friend Guard

Reply:
Yep, and same below